addon/components/snippet-plugin/snippet-list/snippet-list-view.hbs

```diff
@@ -14,7 +14,7 @@
         {{#each @snippetLists as |snippetList|}}
           <tr>
             <td class="selectColumn">
-              <AuControlCheckbox
+              <AuCheckbox
                 @identifier={{snippetList.label}}
```

Comment:
Is this argument still needed?

Reply:
@dkozickis could you answer this please?